example/my_app.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import hydra
+import wandb
+from omegaconf import DictConfig
+
+logger = logging.getLogger(__name__)
+
+
+@hydra.main(config_path="./conf", config_name="config")
+def dummy_training(cfg: DictConfig) -> None:
+    """A dummy function to minimize
+    Minimum is 0.0 at:
+    lr = 0.12, dropout=0.33, db=mnist, batch_size=4
+
+    to execute code --> python my_app.py --multirun +sweeper=wandb
```

Comment:
I think we would prefer overriding default hydra's sweeper instead of doing something like this. For example:

we would be interested in having a plugin which can be invoked as `python my_app.py --multirun hydra/sweeper=wandb`

Reply:
I believe it should still be able to be executed that way no problem. Let me test...

Reply:
The above example was to showcase that you could provide multiple overrides in the [style mentioned in the Hydra docs](https://hydra.cc/docs/patterns/configuring_experiments/). Specifically, you could do something like `python my_app.py --multirun +experiment=[mnist, wandb_sweep]` where the `override /hydra/sweeper: wandb` line would be in `wandb_sweep.yaml` for example. Anyways, I'll test out `python my_app.py --multirun hydra/sweeper=wandb` and report back; I'm pretty sure it should work.

Reply:
[It works](https://wandb.ai/tesfaldet/hydra_plugin_test_project/sweeps/fwd2rxks?workspace=user-tesfaldet) :)

I commented out 
```yaml
defaults:
   - override /hydra/sweeper: wandb
```
from `wandb.yaml` and executed `python my_app.py --multirun hydra/sweeper=wandb +sweeper=wandb` just to see if it'd still use the wandb sweep config defined in `conf/sweeper/wandb.yaml` and it did. If you prefer me to change the example to follow this then I surely can no problem, I just like the example explicitly showing that you can merge override configs with the default config which includes overriding the sweeper.

The code that verifies the type of override is here:
https://github.com/captain-pool/hydra-wandb-sweeper/blob/cbda3c61e9c55d48fb338aec63e4d3e6c275ac91/hydra_plugins/hydra_wandb_sweeper/wandb_sweeper.py#L137-L146

Reply:
Amazing! Thanks again for your wonderful contribution! I'll look into it next week and get back to you shortly. Just as a heads-up, we will be shifting this repository to [github.com/wandb](https://github.com/wandb) and will be making a PR to hydra once everyone is happy about the basic functionalities. So please feel free to fix anything I might have missed. (for example the 2-space yaml indentation problem, you can fix that by modifying the `offset` parameter in `yamlfmt` hook inside [`.pre-commit-config.yaml`](https://github.com/captain-pool/hydra-wandb-sweeper/blob/main/.pre-commit-config.yaml) file. You can find more about it [here](https://github.com/jumanjihouse/pre-commit-hook-yamlfmt/blob/master/README.md).)

Reply:
Awesome! I can make that change in a separate PR :) I'm assuming the repo transfer will include all the commit history?

Reply:
Let's not create a PR right away. I'll talk to my manager and finalize this first. Regarding repo transfer. I'll try my best to transfer all the commit history properly during the PR and the transfer. Also, if something weird happens during the transfer, I'll make sure to highlight your extensive contributions manually in the README.

Reply:
[Fortunately repo transfer should be painless :)](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository)